src/components/Corrections/Corrections.js

```diff
@@ -22,70 +22,75 @@ import KeyInsights from "../KeyInsights";
-      <Chart chartData={populationsChartData} title="Populations" hint="By System" />
-    </ErrorBoundary>
-    <ErrorBoundary placeholder="Unable to render Admissions to Prison. An unhandled error happened. More info could be found in the console.">
-      <Chart chartData={prisonAdmissionsChartData} title="Admissions to Prison" hint="By Type" />
-    </ErrorBoundary>
-    <ErrorBoundary placeholder="Unable to render Parole Revocations chart. An unhandled error happened. More info could be found in the console.">
-      <Chart
-        chartData={paroleRevocationsChartData}
-        title="Post-Release Supervision Revocations"
-        hint="By Type"
-      />
-    </ErrorBoundary>
-    <ErrorBoundary placeholder="Unable to render Probation Revocations Chart. An unhandled error happened. More info could be found in the console.">
-      <Chart
-        chartData={probationRevocationsChartData}
-        title="Probation Revocations"
-        hint="By Type"
-      />
-    </ErrorBoundary>
-    <ErrorBoundary placeholder="Unable to render Releases Chart. An unhandled error happened. More info could be found in the console.">
-      <Chart chartData={releasesChartData} title="Releases" hint="By Type" />
-    </ErrorBoundary>
-    <Sources data={sourceData} />
-  </>
-);
+const Corrections = ({ correctionsData, isAnnual }) => {
+  const {
+    populationsChartData,
+    prisonAdmissionsChartData,
+    paroleRevocationsChartData,
+    probationRevocationsChartData,
+    releasesChartData,
+    flowData,
+    keyInsightsData,
+    flowDiagramLastDate,
+    flowDiagramPrevDate,
+    sourceData,
+  } = correctionsData;
+
+  return (
+    <>
+      <KeyInsights keyInsightsData={keyInsightsData} />
+      <ErrorBoundary placeholder="Unable to render Flow Diagram. An unhandled error happened. More info could be found in the console.">
+        <FlowDiagram
+          data={flowData}
+          lastDate={flowDiagramLastDate}
+          prevDate={flowDiagramPrevDate}
+        />
+      </ErrorBoundary>
+      <ErrorBoundary placeholder="Unable to render Populations Chart. An unhandled error happened. More info could be found in the console.">
+        <Chart
+          annual={isAnnual}
+          chartData={populationsChartData}
+          title="Populations"
+          hint="By System"
+        />
+      </ErrorBoundary>
+      <ErrorBoundary placeholder="Unable to render Admissions to Prison. An unhandled error happened. More info could be found in the console.">
+        <Chart
+          annual={isAnnual}
+          chartData={prisonAdmissionsChartData}
+          title="Admissions to Prison"
+          hint="By Type"
+        />
+      </ErrorBoundary>
+      <ErrorBoundary placeholder="Unable to render Parole Revocations chart. An unhandled error happened. More info could be found in the console.">
+        <Chart
+          annual={isAnnual}
+          chartData={paroleRevocationsChartData}
+          title="Parole Revocations"
```

Comment:
nit: "Post-Release Supervision Revocations" (here and in error place holder)

Reply:
fixed